SafeguardSessions/README.md

```diff
@@ -169,6 +194,17 @@ Distributed under the One Identity - Open Source License. For more information,
 
 <!-- Links -->
 
+[About the project]: #about-the-project
+[Installation and usage]: #installation-and-usage
+[Common errors]: #common-errors
+[Troubleshooting]: #troubleshooting
+[Release policy]: #release-policy
+[Upgrade]: #to-upgrade-the-connector
```

Comment:
I guess section names will be discussed with the TW team to find the best title. I don't really like the idea of starting a section name with "To".

Reply:
In a nutshell: in headings (h1 or h6 level), we don't. 

In sentences that introduce procedure steps (a special kind of sentence in our documentation, with special formatting), we use the following structure: 

_**To &lt;do what the procedure is about&gt;**_ 

(Followed by procedure steps list)

Example from our related standalone guide:

**_To enable the Power BI Desktop application to use the Power BI Connector_**

a, Navigate to ...
(and so on)
 
So if the "To + infinivite" structure is in h1 or h6 headings, let's find an alternative. 
In procedure intro sentences, though, they are recommended at the very least. 

I hope this helps. :-)